os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -0,0 +1,120 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+comment "AMEBASMART Configuration Options"
+
+choice
+	prompt "AMEBASMART Chip Selection"
+	default ARCH_CHIP_RTL8730E
+	depends on ARCH_CHIP_AMEBASMART
+
+config ARCH_CHIP_RTL8730E
+	bool "RTL8730E"
+	select ARCH_CORTEXM33
+	select ARCH_HAVE_FPU
+	select AMEBASMART_WIFI
+	select ARMV8M_HAVE_ICACHE
+	select ARMV8M_HAVE_DCACHE
+

```

Comment:
Hello, could you please update the Kconfig with the appropriate values? I think the older board's values are still left out.

Reply:
Updated, thanks